src-docs/src/views/tables/sorting/sorting_section.js

```diff
@@ -24,7 +24,11 @@ export const section = {
       The following example shows how to configure column sorting via the{' '}
       <EuiCode>sorting</EuiCode>
       property and flagging the sortable columns as{' '}
-      <EuiCode>sortable: true</EuiCode>
+      <EuiCode>sortable: true</EuiCode>.To enable the default sorting ability to
```

Comment:
Typo here (my fault)
```suggestion
      <EuiCode>sortable: true</EuiCode>.To enable the default sorting ability for
```

Also, when there's a line break and then element tags, it gets parsed as not having any space. You then need to add `{' '}` as you'll notice in the previous lines. I'd suggest installing a linter that can auto-fix when you save. I know it helps me a lot.